package.json

```diff
@@ -32,12 +32,18 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-react-hooks": "^2.0.1",
-    "nodemon": "^1.19.2"
+    "nodemon": "^1.19.2",
+    "supertest": "^4.0.2",
```

Comment:
Wher is the test for the query and the endpoint

Reply:
I started the but then found it will be more beneficial to do them after we look at the test database